# Generated by Django 4.2.16 on 2024-11-14 20:58

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Error',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('description', models.TextField()),
                ('received_date', models.DateField()),
                ('severity', models.CharField(max_length=20)),
                ('category', models.CharField(blank=True, max_length=50, null=True)),
                ('source', models.CharField(blank=True, max_length=50, null=True)),
            ],
        ),
        migrations.CreateModel(
            name='Programmer',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('last_name', models.CharField(max_length=50)),
                ('first_name', models.CharField(max_length=50)),
                ('phone', models.CharField(max_length=20)),
            ],
        ),
        migrations.CreateModel(
            name='ErrorFix',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('start_date', models.DateField()),
                ('duration', models.IntegerField()),
                ('cost_per_day', models.DecimalField(decimal_places=2, max_digits=10)),
                ('error', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='app.error')),
                ('programmer', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='app.programmer')),
            ],
        ),
    ]
